libopenage/renderer/stages/camera/manager.h

```diff
@@ -143,6 +144,17 @@ class CameraManager {
 	 * Uniform buffer input for the camera.
 	 */
 	std::shared_ptr<renderer::UniformBufferInput> uniforms;
+
+	/**
+	 * x and z bounds for the camera.
+	 */
+	std::pair<float, float> x_bounds, z_bounds;
```

Comment:
Small nitpick but the docstring should explain more about what these bounds are. Also, you should mention that the first value is the min bound and the second value is the max bound. However, if you follow up on the idea to make this a struct, it would probably not be necessary to document the latter.